### PR TITLE
Lab2

### PR DESCRIPTION
**Pull request by none-word**

### CODE REVIEW

File: app_java/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM openjdk:17
+
+WORKDIR /app_java
+
+COPY . .
```
Comment (AKurmazov) [inline app_java/Dockerfile:5]: nit: I'd suggest copying only needed sources

**Comment by AKurmazov** [comment]
Looks good to me 👍

File: app_java/Dockerfile (hunk 1 above)
Comment (afonyaa) [reply app_java/Dockerfile:5]: I am agreed with Alexander

File: app_java/mvnw [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+#
+# Optional ENV vars
+# -----------------
+#   M2_HOME - location of maven2's installed home dir
+#   MAVEN_OPTS - parameters passed to the Java VM when running Maven
+#     e.g. to debug Maven itself, use
+#       set MAVEN_OPTS=-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8000
+#   MAVEN_SKIP_RC - flag to disable loading of mavenrc files
+# ----------------------------------------------------------------------------
+
```
Comment (afonyaa) [inline app_java/mvnw:36]: Good maven config!

**Comment by afonyaa** [comment]
Nice work!

File: app_java/Dockerfile (hunk 1 above)
Comment (Maxkoz777) [reply app_java/Dockerfile:5]: Still unresolved!

File: app_java/mvnw (hunk 2 above)
Comment (Maxkoz777) [reply app_java/mvnw:36]: I am agreed with Eugenii

**Comment by Maxkoz777** [comment]
Well done, Grisha!
I like your choice for the alternative language)

File: app_java/Dockerfile (hunk 1 above)
Comment (none-word) [reply app_java/Dockerfile:5]: Done

**Comment by Homa3030** [comment]
:+1:

**Pull request closed without merging**